packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -75,10 +75,10 @@ const ProductDetail = ({category, product, isLoading}) => {
     /**************** Wishlist ****************/
     const wishlist = useWishlist()
     // TODO: DRY this handler when intl provider is available globally
-    const handleAddToWishlist = async (quantity) => {
+    const handleAddToWishlist = async (product, quantity) => {
         try {
             await wishlist.createListItem({
-                id: product.id,
+                id: product.id || product.productId,
```

Comment:
What's the difference between `product.id` and `product.productId`? Maybe add a comment explaining why we check both.

Reply:
This `product` is actually either a product or a variant. With variant, its id is called `productId`. I think I need to be more explicit with the wording here.. have both `product` and `variant`.